src/test/py/bazel/bar.py

```diff
@@ -0,0 +1,4 @@
+from src.test.py.bazel import bum
```

Comment:
Files that are used in the tests should be under testdata directory or generated/arranged from within the test. See other tests as an example.